js/aax.js

```diff
@@ -896,6 +896,36 @@ module.exports = class aax extends Exchange {
         return this.parseOHLCVs (data, market, timeframe, since, limit);
     }
 
+    async fetchIndexOHLCV (symbol, timeframe = '1h', since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+            'timeFrame': this.timeframes[timeframe],
+        };
+        const response = await this.publicGetMarketIndexCandles (this.extend (request, params));
+        //
+        //     {
+        //         c: '36592.81000000',
+        //         e: 'BTCUSDT_INDEX@1h_candles',
+        //         h: '36669.29000000',
+        //         l: '36555.28000000',
+        //         o: '36555.83000000',
+        //         s: '1643148000',
+        //         t: '1643148978',
+        //         v: '0.00000000'
+        //     }
+        //
+        return [
+            this.safeTimestamp (response, 't'),
+            this.safeNumber (response, 'o'),
+            this.safeNumber (response, 'h'),
+            this.safeNumber (response, 'l'),
+            this.safeNumber (response, 'c'),
+            this.safeNumber (response, 'v'),
+        ];
+    }
+
```

Comment:
[Look at how we do this for another exchange](https://github.com/ccxt/ccxt/blob/1d29ae43671f79d1a70f9f2e59145765e00c8d62/js/gateio.js#L1746), we give the ability to call `fetchOHLCV` with the param `"price": "index"` in `params`, we should be able to do this on AAX as well